packages/lodestar-cli/src/wallet/Wallet.ts

```diff
@@ -97,10 +99,10 @@ export class Wallet extends Keystore {
       withdrawal: `m/12381/3600/${validatorIndex}/0`,
     };
 
-    const keystores = mapValues(privKeys, (privKey, key) => {
+    const keystores = mapValues(privKeys, async (privKey, key) => {
       const type = key as keyof typeof privKeys;
       const privateKey = PrivateKey.fromBytes(privKey);
-      const keystore = Keystore.encrypt(privKey, emptyPubkey, passwords[type], paths[type]);
+      const keystore = await Keystore.create(passwords[type], privKey, emptyPubkey, paths[type]);
```

Comment:
I think you can replace `emptyPubkey` with `privateKey.toPublicKey().toBytesCompressed()` here, then remove the below comment and `keystore.pubkey` line